requirements.txt

```diff
@@ -1,4 +1,3 @@
 click
 requests
 pyyaml
-statusdb
```

Comment:
OK, I get it now - it won't work doing ``pip install -r requirements.txt`` but it does work to got through ``setup.py`` - I will reverse that change!